docs/userguide/src/site/pages/thredds/ThreddsConfigRef.md

```diff
@@ -235,7 +235,7 @@ The following shows all the configuration options available in the WMS section o
 <WMS>
   <allow>true</allow>
   <allowRemote>false</allowRemote>
-  <paletteLocationDir>/WEB-INF/palettes</paletteLocationDir>
+  <paletteLocationDir>/path/to/custom/palettes</paletteLocationDir>
```

Comment:
I don't think this actually works right now... I need to find the issue that references this

Reply:
Okay, good to know! I see [this comment](https://github.com/Unidata/tds/issues/173#issuecomment-961883903) that it works in the tds content directory so if that is correct we could put that in the docs instead of the absolute path example

Reply:
that's the one! yeah that's the only way it works right now

Reply:
Updated to only mention the relative path